x-pack/plugins/security_solution/public/common/components/exceptions/add_exception_flyout/index.tsx

```diff
@@ -176,14 +176,29 @@ export const AddExceptionFlyout = memo(function AddExceptionFlyout({
 
   useEffect(() => {
     const fetchSingleDataView = async () => {
-      if (dataViewId != null && dataViewId !== '') {
-        const dv = await data.dataViews.get(dataViewId);
+      const notNullDataViewId =
```

Comment:
Hey, I have a couple of questions here.

If I look at the type AddExceptionFlyoutProps, I see` dataViewId?: string; `
meaning that it can be string or undefined, but in the code, you are explicitly checking for null, does there some specific reason for that?

nit: Personally I often found it difficult to read nested ternary expressions, I maybe use for this case something like:
`const notNullDataViewId = dataViewId || maybeRule?.data_view_id || null`

Reply:
I agree the ternary isn't the clearest. I'll try to re-work it. Where exactly am I doing and explicit null check? Having trouble finding that. Looks like all the checks are nullish `!=` not `!==` or were you referencing somewhere else?

Reply:
Oh, I get it! It was `!=` not `!==` :)